processing/src/main/java/org/apache/druid/segment/IndexMergerV9.java

```diff
@@ -1204,12 +1204,18 @@ private File multiphaseMerge(
     List<List<IndexableAdapter>> currentPhases = getMergePhases(indexes, maxColumnsToMerge);
     List<File> currentOutputs = new ArrayList<>();
 
-    log.debug("base outDir: " + outDir);
+    log.debug("Base outDir[%s]", outDir);
 
     try {
       int tierCounter = 0;
       while (true) {
-        log.info("Merging %d phases, tiers finished processed so far: %d.", currentPhases.size(), tierCounter);
+        log.info(
+            "Merging phases[%,d] (indexes[%,d], maxColumnsToMerge[%,d]), tiers finished processed so far[%,d].",
```

Comment:
@kfaraz I think these were the changes we were talking about the other day. This would help us figure out how far along are we in the merging process in Segment generation.